lib/middleware/routes-nunjucks/routes-nunjucks.js

```diff
@@ -8,36 +8,63 @@ const {
   formatProperties
 } = require('~/fb-runner-node/page/page')
 
-const nunjucksRouter = () => {
-  router.use(/^\/([^.]+)$/, (req, res, next) => {
+const USERDATA = {
```

Comment:
This was defined below at line 17 but it was missing some methods (in other words, if `req.user` was not _defined_ then using the previous version of this object instead would cause an error)